sdk/script/management-sdk.ts

```diff
@@ -212,9 +206,9 @@ class AccountManager {
             .then((res: JsonResponse) => res.body.history);
     }
 
-    public release(appName: string, deploymentName: string, fileOrPath: File | string, targetBinaryVersion: string, description?: string, isMandatory: boolean = false, uploadProgressCallback?: (progress: number) => void): Promise<void> {
+    public release(appName: string, deploymentName: string, fileOrPath: File | string, targetBinaryVersion: string, description?: string, rollout?: number, isMandatory: boolean = false, uploadProgressCallback?: (progress: number) => void): Promise<void> {
```

Comment:
Did you want to convert the description, mandatory and rollout options into an "updateMetadata" parameter like you did for `patchRelease` and `promote`?